plugins/org.python.pydev.shared_core/src/org/python/pydev/shared_core/IMiscConstants.java

```diff
@@ -33,6 +33,9 @@
     String MYPY_PROBLEM_MARKER = "org.python.pydev.mypyproblemmarker";
     String MYPY_MESSAGE_ID = "mypy_message_id";
 
+    String FLAKE8_PROBLEM_MARKER = "org.python.pydev.flake8problemmaker";
```

Comment:
Note: you also need to register the `flake8problemmaker` in the `org.python.pydev/plugin.xml` (search for `mypyproblemmarker`) in that file.